examples/all-clusters-app/ameba/main/chipinterface.cpp

```diff
@@ -142,6 +142,23 @@ static void InitServer(intptr_t context)
 #endif
 }
 
+extern "C" bool matter_server_is_commissioned()
+{
+    return (chip::Server::GetInstance().GetFabricTable().FabricCount() != 0);
+}
+
+extern "C" void matter_get_fabric_indexes(uint16_t *pFabricIndexes)
```

Comment:
How does the caller know how much memory they have to provide?  How do we make sure we are not writing out of bounds on the provided buffer?